uni/lib/main.dart

```diff
@@ -175,6 +175,7 @@ class MyAppState extends State<MyApp> {
       builder: (context, themeNotifier, _) => MaterialApp(
         title: 'uni',
         theme: applicationLightTheme,
+        debugShowCheckedModeBanner: false, // this here sanat
```

Comment:
Any reason to disable the debug banner?

Reply:
Hiding the debugbanner when the app is in production is best practice. I therefore made it untrue.
Is i am not supposed to change it?

Reply:
Firstly, this is not related to the issues you are targeting. Anyway, in production mode changing this has no effect, and in debug is desirable to be shown, so there's no need to do this

Reply:
The banner is only shown in **debug** mode, per [Flutter's documentaion](https://api.flutter.dev/flutter/material/MaterialApp/debugShowCheckedModeBanner.html), we prefer to keep it that way because it's a nice visual indicator that the app is in debug mode, if you could revert this it would be nice 😅

Reply:
okok Sure.